TCL4 Data Management/utm-tcl4-dmp-con.yaml

```diff
@@ -100,9 +100,6 @@ definitions:
     type: object
     required:
       - "metaData"
-      - "plannedBvlosLandingPoints"
-      - "plannedVlosLandingPoints"
```

Comment:
Why are we making this optional?  thanks

Reply:
This is ConPreRunOp, so this should be submitted per op. Some ops may not have BVLOS landing points and others may not have VLOS landing points.  (according to Hemil, Edgard, and Brandon).